From 2bb92cf88cd3af3d1a969bf5afa37fe2d59d247e Mon Sep 17 00:00:00 2001
From: mselva006c <mani_selvaraj@comcast.com>
Date: Fri, 29 May 2020 16:46:32 +0000
Subject: [PATCH] RDK-28776: Yocto 3.1 Bootup time is longer

        Patch systemd-timesyncd source code inorder to provide the event to XRE receiver.
        Verify multi target builds with sd/se is fine
Source: COMCAST
License: LGPL-2.1-or-later
Upstream-Status: Pending
Signed-off-by: mselva006c <mani_selvaraj@comcast.com>
---
 src/timesync/timesyncd-manager.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/timesync/timesyncd-manager.c b/src/timesync/timesyncd-manager.c
index 7b71e98f56..561f35462c 100644
--- a/src/timesync/timesyncd-manager.c
+++ b/src/timesync/timesyncd-manager.c
@@ -301,6 +301,8 @@ static int manager_adjust_clock(Manager *m, double offset, int leap_sec) {
         /* If touch fails, there isn't much we can do. Maybe it'll work next time. */
         (void) touch("/var/lib/systemd/timesync/clock");
         (void) touch("/run/systemd/timesync/synchronized");
+        log_info("Time response event, set the event flag");
+        (void) touch("/tmp/clock-event");
 
         m->drift_freq = tmx.freq;
 
